requirements_test.txt

```diff
@@ -1,7 +1,10 @@
 # linters such as flake8 and pylint should be pinned, as new releases
 # make new things fail. Manually update these pins when pulling in a
 # new version
+
+# When updating this file, update .pre-commit-config.yaml too
 asynctest==0.13.0
+black==19.3b0
```

Comment:
b0 sounds like beta0, should this be rather pinned to some released version?

Reply:
All the existing releases are alphas or betas:
https://pypi.org/project/black/#history